### PR TITLE
docs: minor changes for RFC-0150/Wallets

### PR DESCRIPTION
**Pull request by agubarev**
Description
---
Added table of contents and a few very minor changes.

Motivation and Context
---
Iterative review of existing RFCs.

How Has This Been Tested?
---

### CODE REVIEW

File: src/RFC-0150_Wallets.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
 # RFC-0150/Wallets
+![status: draft](https://github.com/tari-project/tari/raw/master/RFC/src/theme/images/status-draft.svg)
```
Comment (sdbondi) [inline src/RFC-0150_Wallets.md:2]: Can this be stabilised?

File: src/RFC-0150_Wallets.md [hunk 2]
```diff
@@ -61,58 +73,58 @@ This RFC is derived from a proposal first made in [this issue](https://github.co
...
-- It MUST be able to reconstruct the wallet state from a manually entered master seed key. 
-- It MUST have a mechanism to systematically search through the Tari blockchain and mempool for unspent and unconfirmed 
-transactions, using the keys derived from the master key.
-- The master seed key SHOULD be derivable from a specific set of mnemonic word sequences using known word lists.
-- It MAY enable the reconstruction of the master seed key by scanning a coded marker of a paper wallet.
+- Wallet **MUST** be able to reconstruct the wallet state from a manually entered master seed key. 
+- Wallet **MUST** have a mechanism to systematically scan through the Tari blockchain and [mempool](RFC-0190_Mempool.md) for `Unspent` and `Unconfirmed` 
+transactions, using keys derived from the master key.
+- The master seed key **SHOULD** be derivable from a set of mnemonic word sequences using known word lists.
+- Wallet **MAY** enable the reconstruction of the master seed key by scanning a coded marker of a paper wallet.
```
Comment (sdbondi) [inline src/RFC-0150_Wallets.md:128]: Needs a changelog

File: src/RFC-0150_Wallets.md [hunk 3]
```diff
@@ -61,58 +73,58 @@ This RFC is derived from a proposal first made in [this issue](https://github.co
...
+- Wallet **SHOULD** be able to generate derived, transactional, cryptographic keypairs from the master seed key using deterministic 
+keypair generation.
+- Wallet **SHOULD** store wallet state using a password or passphrase encrypted persistent key-value database.
+- Wallet **SHOULD** provide the ability to store backup of the wallet state to a single encrypted file to simplify wallet recovery and 
 reconstruction at a later stage.
-- It MAY provide the ability to export the master seed key or wallet state as a printable paper wallet, using coded markers.
+- Wallet **MAY** provide the ability to export the master seed key or the wallet state as a printable paper wallet, using coded markers.
 
-#### Different Methods for Recovering Wallet State of Tari Software Wallet
+#### Tari Software Wallet: State Recovery Methods
```
Comment (sdbondi) [inline src/RFC-0150_Wallets.md:121]: I don't think we have a full detailed description of the wallet recovery (might be mistaken).
@stringhandler Do you think we should include a full description here? This RFC is probably intentionally very light on details so maybe we need a new RFC describing wallet recovery. We have bulletproof rewinding but not  the exact wallet recovery process. 

```suggestion
#### State Recovery
```

File: src/RFC-0150_Wallets.md [hunk 4]
```diff
@@ -1,10 +1,23 @@
 # RFC-0150/Wallets
+![status: stable](https://github.com/tari-project/tari/raw/master/RFC/src/theme/images/status-stable.svg)
+**Maintainer(s)**: [Yuko Roodt](https://github.com/neonknight64), [Cayle Sharrock](https://github.com/CjS77)
```
Comment (CjS77) [inline src/RFC-0150_Wallets.md:3]: Don't change the location of the status badge

**Pull request closed without merging**